readinglist/views/article.py

```diff
@@ -56,6 +57,7 @@ class ArticleSchema(ResourceSchema):
     class Options:
         readonly_fields = ('url', 'stored_on') + \
             ResourceSchema.Options.readonly_fields
+        unique_fields = ('url', 'resolved_url')
```

Comment:
The unicity on ``resolved_url`` seems a bit dangerous to me. Also most of the url will have an empty ``resolved_url`` to start with until the worker populates it.